SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/OptionTests.swift

```diff
@@ -15,34 +15,34 @@ class OptionTests: XCTestCase {
     func testSwiftCallRustOptionPrimitive() throws {
```

Comment:
Autoformatting did something to the spacing in this whole file... I'll leave a comment where I actually made changes